### PR TITLE
Remove new-user link

### PR DESCRIPTION
**Pull request by whaught**
Issue https://github.com/google/exposure-notifications-verification-server/issues/304

## Proposed Changes

* Remove the link to create a new user
    * New users should be invited by an existing admin and will receive an email.
    * I'll remove the page entirely in the future. An oncaller can still help someone directly navigate to login/create if we see any problems with the email-invite flow.

### CODE REVIEW

**Comment by mikehelmick** [comment]
Should we leave this as is for at least a full release? Check this in after v0.8 is cut?

**Comment by whaught** [comment]
> Should we leave this as is for at least a full release? Check this in after v0.8 is cut?

We could just leave it for a bit. The idea here was to kill the link, but leave the page up. Then if someone screams one can still manually navigate there. Alternatively we can leave as-is for a release and kill the whole thing

**Comment by mikehelmick** [comment]
> > Should we leave this as is for at least a full release? Check this in after v0.8 is cut?
> 
> We could just leave it for a bit. The idea here was to kill the link, but leave the page up. Then if someone screams one can still manually navigate there. Alternatively we can leave as-is for a release and kill the whole thing

we won't know if someone screams :)

let's 
/hold
for a week

**Comment by whaught** [comment]
/reopen

didn't mean to close this one

**Comment by shohansp** [comment]
Suported

File: pkg/controller/user/delete.go
Comment (sethvargo) [inline pkg/controller/user/delete.go:62]: Bad rebase?

Comment (sethvargo) [reply pkg/controller/user/delete.go:62]: Actually, looks like that quote shouldn't be anywhere in this string.

**Comment by mikehelmick** [comment]
this is fine to check in now

I'll let you address Seth's comment first

File: pkg/controller/user/delete.go
Comment (whaught) [reply pkg/controller/user/delete.go]: Sorry this is kind of unrelated, just saw the message at one point while testing and didn't like the weird quotes

**Review by mikehelmick — APPROVED** [review]

**Pull request merged**
